mlmodel/docs/Format/Model.rst

```diff
@@ -6,6 +6,7 @@ and can be any one of the following types:
 
 Neural Networks
   - ``NeuralNetwork``
+  - ``mlprogram``
```

Comment:
`mlprogram` is the name of the member instance. `MILSpec.Program` is the type name. You want `MILSpec.Program`; all the other names here are type names not instance member names.

I'm also recommend moving it above `NeuralNetwork` since it's more important.